src/rdma_van.h

```diff
@@ -125,26 +126,39 @@ class RDMAVan : public Van {
     struct sockaddr_in addr;
```

Comment:
I think for IPv6 we need a `struct sockaddr_in6 addr6;`

Reply:
may need sockaddr_storage for better compatibility later, ref: http://long.ccaba.upc.edu/long/045Guidelines/eva/ipv6.html